### PR TITLE
Feature: Added a splash screen

### PR DESCRIPTION
**Pull request by 0x5bfa**
### PR Checklist

- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Related #4180
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### Screenshots

_None_

### CODE REVIEW

**Comment by yaira2** [comment]
@0x5bfa this is working really well 🎉

Is the mica backdrop broken?

**Comment by 0x5bfa** [comment]
Mica is disabled in displaying splash screen because I wanna the app to load minimum components; however, content page's backdrop is still enabled.

**Comment by yaira2** [comment]
@0x5bfa that's expected but it doesn't work for me at all, even after the app is fully loaded.

**Comment by 0x5bfa** [comment]
OMG I made a terrible mistake, I can fix this.

File: src/Files.App/MainWindow.xaml.cs [hunk 1]
```diff
@@ -19,20 +18,23 @@ namespace Files.App
...
 
 		public IntPtr WindowHandle { get; }
 
-		private MainPageViewModel mainPageViewModel;
-
-		private IApplicationService ApplicationService { get; } = Ioc.Default.GetRequiredService<IApplicationService>();
-
 		private MainWindow()
 		{
+			ApplicationService = new ApplicationService();
```
[the remark this answers is not in the document]
Comment (0x5bfa) [reply src/Files.App/MainWindow.xaml.cs:32]: This class is instanced before the host is initialized. I need to do this instead of injecting.

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**